client/src/components/Logo/Logo.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./Logo.css";
+
+// Function based React Component
+const Logo = props => {
+  // Default Class to apply to Component
+  let classList = `Logo`;
+  if (props.small) {
+    classList += ` smallLogo`;
```

Comment:
Can you change this class name to `small-logo`? We generally want our classes to be kebab case